docs/en/dgs/tutorial.md

```diff
@@ -33,81 +35,53 @@ The generated data are stored in `/tmp/u2i_gen/training` and `/tmp/u2i_gen/strea
 
 ## 2. Train model offline
 
-FIXME(@Seventeen17): refactor this section
 ```shell
-cd graphlearn/examples/tf/ego_sage
-python train_supervised.py
+cd graphlearn/examples/tf/ego_bipartite_sage
+python train.py
 ```
 
 Ref to [GraphLearn-Training](../gl/intro.md) for more details.
 
 
 ## 3. Export TF SavedModel
 
-FIXME(@Seventeen17): refactor this section
 
 First, export model as tf SavedModel, we need to filter some of the placeholders as model serving inputs based on the
 computational graph, you can view the computational graph with the help of Tensorboard to determine the inputs for the
 serving subgraph.
 
-The offline training model is saved in `graphlearn/examples/tf/ego_sage/ckpt`, and we save the final serving model in
-`./ego_sage_sup_model` directory, the inputs to the subgraphs are the placeholders `0,2,3`.
+The offline training model is saved in `graphlearn/examples/tf/ego_bipartite_sage/ckpt`, and we save the final serving model in
+`graphlearn/examples/tf/serving/ego_bipartite_sage` directory, the inputs to the user-subgraph are the placeholders `0,3,4` according to TensorBoard.
```

Comment:
we should consider better apporach.

Reply:
Yes, but for now on, I haven't find a more appropriate solution, I will provide more automated tool in subsequent commits.